### ISSUE TITLE
Write an email prompt for users to change password 

### ISSUE THREAD

**Issue by Jo-Dwyer** [cityofaustin/techstack#4464]
Joplin users should change their passwords to a new and unique password. 
- [ ] login
- [ ] go to account settings
- [ ] change password
- [ ] enter old password 
- [ ] enter a new password

![Screen Shot 2020-06-04 at 10.43.21 AM.png](https://images.zenhubusercontent.com/5d641145b9bd2f17c566338c/923daec7-22af-43b8-8eed-1050ac31165e)

Good to know: Currently, the 'Forgot password' button is not yet functional. If a user has forgotten their password, it can be reset by an admin in ODD. The user must then login and reset this password themselves to ensure security.

**Comment by sammodabber** [comment]
@Niicck Is the email in place for this issue to be relevant next sprint?

**Comment by Niicck** [comment]
Still waiting on CTM to finish setting up the email. So hopefully next sprint.

**Comment by sammodabber** [comment]
@Niicck Was the email ever set up?

**Comment by Niicck** [comment]
Yep! Our login flow was finished by another issue.
